packages/amplify_auth_plugin_interface/lib/src/Password/ConfirmPasswordRequest.dart

```diff
@@ -13,27 +13,27 @@
  * permissions and limitations under the License.
  */
 
-import './PasswordOptions.dart';
+import 'ConfirmResetPasswordOptions.dart';
+import 'ConfirmResetPasswordRequest.dart';
 
-class ConfirmPasswordRequest {
+/// Encapsulates parameters for a confirm reset password request
+@Deprecated("Please use 'ConfirmResetPasswordRequest' instead")
```

Comment:
nit: should use backtick to wrap class name? Ditto other deprecation annotation

Reply:
I couldn't find concrete standards on formatting deprecated messages. I just took a quick look at the flutter/flutter repo and it looks like they do not wrap classes or function in any symbols. (example: https://github.com/flutter/flutter/blob/e7b7ebc066c1b2a5aa5c19f8961307427e0142a6/packages/flutter/lib/src/material/raised_button.dart#L27-L30). I will update all the messages to follow that.

Reply:
```suggestion
@Deprecated('Please use ConfirmResetPasswordRequest instead')
```